src/current/_includes/common/numa_and_go.md

```diff
@@ -0,0 +1,8 @@
+In a [NUMA (non-uniform memory access) architecture](https://en.wikipedia.org/wiki/Non-uniform_memory_access), the system’s memory is physically distributed across multiple memory banks or "nodes", and each node is assigned to a processor. A processor can access its local memory much faster than non-local memory. This non-uniform memory access can lead to performance differences depending on data location.
+
+CockroachDB is written in Go, which has no support for NUMA scheduling or pinning. To run multiple CockroachDB clusters on a NUMA architecture:
+
+- Assign no more than 32 cores to the node for each instance.
+- Ensure that your orchestration framework, process manager, or startup scripts start each CockroachDB in a separate NUMA node.
+- Use the `--locality` flag to indicate when multiple CockroachDB instances are running on a given physical host.
+- Ensure that each CockroachDB cluster writes to a unique set of storage volumes (block devices).
```

Comment:
I don't understand this last recommendation relative to the rest of the recommendations and would remove it from this PR.  i.e., why is the storage volume important here?  I don't think we've made any recommendations about aligning storage devices and NICs to the same NUMA domain, so I don't want to expand the scope of these guidelines into general best practices for NUMA.

Reply:
I took all of this straight from what @nicktrav had there. Maybe he can weigh in.